### PR TITLE
[WEB-2714] Set patient MRN and other details while accepting invite to clinic

### PR DESCRIPTION
**Pull request by clintonium-119**
See [WEB-2714]

Supports: https://github.com/tidepool-org/blip/pull/1331

[WEB-2714]: https://tidepool.atlassian.net/browse/WEB-2714?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by krystophv — APPROVED** [review]
Couple stylistic suggestions, but nothing holding back approval 👍🏼

File: clinics.js [hunk 1]
```diff
@@ -402,14 +402,19 @@ module.exports = function (common) {
...
+     * @param {Object} [patientDetails] - Patient details to set for newly created clinic user
+     * @param {String} [patientDetails.fullName] - The full name of the patient
+     * @param {String} [patientDetails.birthDate] - YYYY-MM-DD
+     * @param {String} [patientDetails.mrn] - The medical record number of the patient
+     * @param {String[]} [patientDetails.tags] - Array of string tag IDs
      * @param {Function} cb
      * @returns {cb} cb(err, response)
     */
-    acceptPatientInvitation: function(clinicId, inviteId, cb){
+    acceptPatientInvitation: function(clinicId, inviteId, patientDetails = null, cb){
```
[the remark this answers is not in the document]
Comment (clintonium-119) [reply clinics.js:413]: Agree.  We can pass `undefined` for `patientDetails` explicitly when we need to pass a cb.
Will update

File: clinics.js [hunk 2]
```diff
@@ -402,14 +402,19 @@ module.exports = function (common) {
...
       common.assertArgumentsSize(3);
```
[the remark this answers is not in the document]
Comment (clintonium-119) [reply clinics.js:414]: Yep.  Good catch

**Comment by clintonium-119** [comment]
@krystophv Can you please have a look at https://github.com/tidepool-org/platform-client/pull/161/commits/b22b48135d671a80b3952c85b3c06048c12170db and re-approve?

I ended up using the same convention we use for other handlers where the last argument prior to the callback is optional.
For ref: [getPatientsForClinic](https://github.com/tidepool-org/platform-client/blob/develop/clinics.js#L236C5-L241)

**Review by krystophv — APPROVED** [review]
Update LGTM 💯

**Pull request merged**